Paco-Server/git-hooks/pre-commit

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+source ~/.bash_profile
```

Comment:
Do there need to be specific statements in the .bash_profile to support this script?

Also, it shouldn't be necessary to source ~/.bash_profile since this will be run from the commandline where .bash_profile was sourced during startup of the commandline.